custom/icds_reports/utils/aggregation_helpers/mbt.py

```diff
@@ -247,7 +247,7 @@ def query(self):
         LEFT JOIN "{person_cases_ucr}" mother on mother.doc_id=t.mother_case_id
           AND awc.state_id = mother.state_id
           AND lower(substring(mother.state_id, '.{{3}}$'::text)) = '{state_id_last_3}'
-        LEFT JOIN "ccs_record_monthly_{month}" ccs on ccs.person_case_id=mother.doc_id AND ccs.add=t.dob
+        LEFT JOIN "ccs_record_monthly_{month}" ccs on ccs.person_case_id=mother.doc_id AND ccs.add=t.dob AND ccs.child_name=t.person_name
```

Comment:
This means that all ccs_record info will be empty until child_name is populated. I think that's ok for MBT, but want to verify

Reply:
Yeah, I think that is fine. This is specifically to deal with what happened when there are multiple ccs cases with matching DOB which was causing duplicate rows in the join (generally twins registered after birth, or more likely based on the frequency, children registered with the same DOB because actual one is unknown). Your earlier question on twins raised a good point that I am investigating, but I think missing a few matches is better than the duplication, especially since we said we often cant match